src/helpers/sync_helpers.sh

```diff
@@ -18,11 +18,11 @@ function sync_branch_steps {
       echo "merge_tracking_branch"
       echo "merge $(parent_branch "$branch")"
     else
-      echo "rebase_tracking_branch"
+      echo "$PULL_BRANCH_STRATEGY""_tracking_branch"
 
       if [ "$branch" = "$MAIN_BRANCH_NAME" ] && [ "$(has_remote_upstream)" = true ]; then
         echo "fetch_upstream"
-        echo "rebase upstream/$MAIN_BRANCH_NAME"
+        echo "$PULL_BRANCH_STRATEGY upstream/$MAIN_BRANCH_NAME"
```

Comment:
Thinking about this some more, I'm now agreeing with @charlierudolph. The main branch should never be merged. (1) it only has a single commit on it, so there is no difference between merge and rebase, and (2) having merges on the main branch is ugly, and one of the goals of GT is to prevent exactly that.